dashboard/dashboard-front/src/components/docs/DocsPlugin.vue

```diff
@@ -0,0 +1,44 @@
+<script setup lang="ts">
+import { ref, type Ref, onMounted } from 'vue'
+import { toastService } from '@/services/ToastService'
+import { apiClient } from '@/services/ApiClient'
+import { useRoute } from 'vue-router'
+
+const route = useRoute()
+const pageName = route.params.pageName
+
+const docPageContent: Ref<PageContent> = ref({
+    doc_name: '',
+    html_content: '',
+})
+
+interface PageContent {
+    doc_name: string
+    html_content: string
+}
```

Comment:
Duplicate of the same interface from `DocsPage.vue`. file I would consider extracting them to a separate file.